edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/linkhandlers/LinkHandlerProvider.cs

```diff
@@ -63,35 +71,64 @@ public ILinkHandler Create(IAmqpLink link, Uri uri)
 
         internal ILinkHandler GetLinkHandler(LinkType linkType, IAmqpLink link, Uri uri, IDictionary<string, string> boundVariables)
         {
-            switch (linkType)
+            if (linkType == LinkType.Cbs)
+            {
+                return CbsLinkHandler.Create(link, uri);
```

Comment:
This is because CBS links don't have identity?

Reply:
Yes, right.